internal/db/schema/migrations/oss/postgres/100/02_iam_role_org.up.sql

```diff
@@ -0,0 +1,139 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+ 
+  -- Create the enumeration table for the grant scope types for the org iam_role
+  create table iam_role_org_grant_scope_enm (
+    name text primary key
+      constraint only_predefined_scope_types_allowed
+        check(
+          name in (
+            'children',
+            'individual'
+          )
+        )
+  );
+  comment on table iam_role_org_grant_scope_enm is
+  'iam_role_org_grant_scope_enm is an enumeration table for role grant scope types for the iam_role_org table.';
+
+  -- Insert the predefined grant scope types for iam_role_org
+  insert into iam_role_org_grant_scope_enm (name)
+  values
+    ('children'),
+    ('individual');       
+
+  create table iam_role_org (
+    public_id wt_role_id primary key
+      constraint iam_role_fkey
+        references iam_role(public_id)
+        on delete cascade
+        on update cascade,
+    scope_id wt_scope_id not null
+      constraint iam_scope_org_fkey
+        references iam_scope_org(scope_id)
+        on delete cascade
+        on update cascade,
+    name text,
+    description text,
+    grant_this_role_scope boolean not null default false,
+    grant_scope text not null
+      constraint iam_role_org_grant_scope_enm_fkey
+        references iam_role_org_grant_scope_enm(name)
+        on delete restrict
+        on update cascade,
+    version wt_version,
+    grant_this_role_scope_update_time wt_timestamp,
+    grant_scope_update_time wt_timestamp,
+    create_time wt_timestamp,
+    update_time wt_timestamp,
+    unique(public_id, grant_scope)
```

Comment:
Can you name this unique constraint with a name that follows our standard naming convention for unique constraints? Our unique constraint naming convention is to concatenate the table name with each column in the unique constraint and then add the suffix `uq`. (ex: `table_name_col1_col2_colN_uq`)

**Example** shows a foreign key constraint and a unique constraint:

```sql
create table widget_shelf (
  public_id wt_public_id primary key,
  scope_id wt_scope_id not null
    constraint iam_scope_fkey
      references iam_scope (public_id)
      on delete cascade
      on update cascade,
  name wt_name,
  constraint widget_shelf_scope_id_name_uq
    unique(scope_id, name)
);
```

Also, consider reversing the order of the columns